eslint.config.js

```diff
@@ -0,0 +1,444 @@
+	"react/boolean-prop-naming": ERROR,
+	"react/button-has-type": ERROR,
+	"react/default-props-match-prop-types": ERROR,
+	"react/forbid-component-props": TRANSITION_IGNORE,
+	"react/forbid-foreign-prop-types": ERROR,
+	"react/jsx-boolean-value": ERROR,
+	"react/jsx-closing-bracket-location": [ERROR, "tag-aligned"],
+	"react/jsx-closing-tag-location": ERROR,
+	"react/jsx-curly-brace-presence": ERROR,
+	"react/jsx-curly-spacing": [
+		ERROR,
+		{
+			children: true,
+		},
+	],
+	"react/jsx-equals-spacing": ERROR,
+	"react/jsx-first-prop-new-line": ERROR,
+	"react/jsx-handler-names": ERROR,
+	"react/jsx-indent-props": [ERROR, "tab"],
+	"react/jsx-no-bind": [
+		ERROR,
+		{
+			ignoreRefs: true,
+		},
+	],
+	"react/jsx-no-literals": TRANSITION_IGNORE,
+	"react/jsx-one-expression-per-line": TRANSITION_IGNORE,
+	"react/jsx-pascal-case": ERROR,
+	"react/jsx-sort-props": [
+		ERROR,
+		{
+			callbacksLast: true,
+			ignoreCase: false,
+			shorthandFirst: true,
+		},
+	],
+	"react/jsx-tag-spacing": [
+		ERROR,
+		{
+			beforeSelfClosing: "never",
+		},
+	],
+	"react/jsx-wrap-multilines": ERROR,
+	"react/no-access-state-in-setstate": ERROR,
+	"react/no-array-index-key": TRANSITION_WARNING,
+	"react/no-danger": TRANSITION_WARNING,
+	"react/no-did-mount-set-state": ERROR,
+	"react/no-did-update-set-state": ERROR,
+	"react/no-direct-mutation-state": ERROR,
+	"react/no-multi-comp": [
+		ERROR,
+		{
+			ignoreStateless: true,
+		},
+	],
+	"react/no-redundant-should-component-update": ERROR,
+	"react/no-set-state": TRANSITION_IGNORE,
+	"react/no-typos": ERROR,
+	"react/no-unused-prop-types": ERROR,
+	"react/no-unused-state": TRANSITION_WARNING,
+	"react/no-will-update-set-state": ERROR,
+	"react/prefer-es6-class": [ERROR, "always"],
+	"react/prefer-stateless-function": ERROR,
+	"react/require-default-props": [
+		ERROR,
+		{
+			forbidDefaultForRequired: true,
+		},
+	],
+	"react/self-closing-comp": ERROR,
+	"react/sort-comp": ERROR,
+	"react/sort-prop-types": [
+		ERROR,
+		{
+			callbacksLast: false,
+			ignoreCase: false,
+			requiredFirst: false,
+			sortShapeProp: true,
+		},
+	],
+	"react/style-prop-object": ERROR,
+	"react/void-dom-elements-no-children": ERROR,
+};
+
+const es6ImportRules = {
+	"import/first": ERROR,
+	"import/newline-after-import": [
+		WARNING,
+		{
+			count: 2,
+		},
+	],
+	"import/no-absolute-path": ERROR,
+	"import/no-amd": ERROR,
+	"import/no-commonjs": ERROR,
+	"import/no-duplicates": ERROR,
+	"import/no-dynamic-require": TRANSITION_WARNING,
+	"import/no-extraneous-dependencies": ERROR,
+	"import/no-internal-modules": [
+		ERROR,
+		{
+			allow: [
+				"**/src/**",
+				"**/lib/**",
+				"**/data/**",
+				"**/config/**",
+				"**/test/**",
+				"react-dom/server",
+				"**/react-select/*",
+			],
+		},
+	],
+	"import/no-mutable-exports": ERROR,
+	"import/no-named-as-default": ERROR,
+	"import/no-named-as-default-member": ERROR,
+	"import/no-named-default": ERROR,
+	"import/no-unassigned-import": ERROR,
+};
+
+module.exports = [
```

Comment:
Forgot one :sweat_smile: 
```suggestion
export default [
```